### ISSUE TITLE
Surface default JAVA_HOME

### ISSUE THREAD

**Issue by jvilk** [jvilk/locate-java-home#1]
Right now, we don't expose which `JAVA_HOME` is the default on the given platform.

While my projects do not require this, it might be nice. If you're using my library, leave a note here and I'll likely implement it (unless you want to take the initiative!).

**Comment by maccelerated** [comment]
I guess it's possible to have multiple versions of a JDK installed on Windows too. I'm somewhat curious about sorting by build or just having that info exposed. The installed JRE does seem to list first.

**Comment by jvilk** [comment]
@maccelerated Yeah, multiple versions are possible. At one point (maybe it's still true?), it was normal to have many versions installed concurrently on Windows.

The version information should be included in the JDKs returned to you by the library. Is build information conveyed in the version of the JDKs returned to you?

**Comment by maccelerated** [comment]
I believe it's only in the JDK's path, not in the version of the home result. I could just pull it out with regex, I suppose.

**Comment by jvilk** [comment]
The security number, which we discussed earlier (incorrectly) as the build number, is now exposed in the information returned by locate_java_home.

Note that Java 9 uses more sensible version schemes where the security number is actually encoded within the version number, so these will have a security number of '0'. See http://openjdk.java.net/jeps/223 and https://blogs.oracle.com/java-platform-group/a-new-jdk-9-version-string-scheme for more info.

Who knew JDK/JRE versions were so complicated? :)
